config/default.schema.json

```diff
@@ -1624,6 +1624,11 @@
 			"type": "number",
 			"default": 10000,
 			"description": "Size of chunk for synchronization"
+		},
+		"FEATURE_MEDIA_SHELF_ENABLED": {
```

Comment:
Do we need seed data for this feature? Maybe a existing board for one user?

Reply:
I would say no, since the data for the board is very dynamic and depends on what tools exist in the seed data or how the environment is created and it should create a media shelf as soon as you enter the page anyways.